profile/README.md

```diff
@@ -1 +1,9 @@
-RITS is dead. Long live [ARC](https://github.com/UCL-ARC).
+Research IT Services (RITS) was the first team of research software engineers and research computing specialists in University College London.
```

Comment:
Just a thought, not pushing it
```suggestion
Research IT Services (RITS) was the first team of research technical professionals in University College London.
```

Reply:
I think I'd stick with the more verbose description, as there were other technical teams with different specialisms in existence, just not centrally for the whole university with an RSE team too.